from wave_pay.exceptions.flutterwave_error import FlutterWaveError

class InvalidChallengeError(FlutterWaveError):
    pass
